.github/workflows/ci.yml

```diff
@@ -36,6 +36,10 @@ jobs:
       - name: Install build deps
         shell: bash
         run: |
+          curl -LO https://apache.jfrog.io/artifactory/arrow/ubuntu/apache-arrow-apt-source-latest-$(grep -F VERSION_CODENAME /etc/os-release | cut -d= -f2).deb
```

Comment:
why, do we need the arrow source? Is it for the build with bindgen?

Reply:
It's not the source, it's the APT repository: https://arrow.apache.org/install/. We need it because otherwise `apt update` fails with a GPG error on some GDAL images.

Reply:
well then i guess we have no choice...